### ISSUE TITLE
Investigate psa-prod extended runtime

### ISSUE THREAD

**Issue by alexdunnjpl** [NASA-PDS/registry-sweepers#123]
### 💡 Description

<img width="1335" alt="image" src="https://github.com/NASA-PDS/registry-sweepers/assets/75815303/7fe03c68-175b-4e0a-8417-4c0c7e200bc4">

Killing the latest tasks for now - will loop back later

**Comment by alexdunnjpl** [comment]
schedule disabled until this issue is resolved

**Comment by alexdunnjpl** [comment]
Blocked by #120

**Comment by jordanpadams** [comment]
duplicate of https://github.com/NASA-PDS/registry-sweepers/issues/162
